torch/fx/experimental/proxy_tensor.py

```diff
@@ -454,6 +454,7 @@ def assert_proxy_tensor(e):
 
 
 SymInt = torch._C.SymIntNode
+SymFloat = torch._C.SymFloatNode
```

Comment:
We dump these in the torch. namespace since they don't look private, so you can access them directly on `torch.SymFloatNode` !
See other comment about making these private or documenting them.